cypress/e2e/04-covid_only_tests.cy.js

```diff
@@ -23,9 +23,10 @@ describe("Conducting a COVID test from:", () => {
   const lastName = patient.lastName;
   const covidOnlyDevice = generateCovidOnlyDevice();
   const queueCard = "[data-cy=prime-queue-item]:last-of-type";
-  loginHooks();
 
   before("setup spec data", () => {
+    loginHooks();
```

Comment:
also for spec4
<img width="855" alt="Screenshot 2024-03-20 at 13 51 12" src="https://github.com/CDCgov/prime-simplereport/assets/20211771/8e29bccf-ed9e-430c-b8f6-32188138eda3">